projects/dotcms-field-elements/src/components/dot-textfield/dot-textfield.tsx

```diff
@@ -49,6 +53,33 @@ export class DotTextfieldComponent {
     componentWillLoad(): void {
         this.status = getOriginalStatus(this.isValid());
         this.emitStatusChange();
+        this.validateProps();
+    }
+
+    @Watch('disabled')
+    disabledWatch(_newValue, oldValue): void {
+        this.disabled = dotPropValidator(
+            {
+                fieldType: 'dot-textfield',
+                fieldName: this.name,
+                name: 'disabled',
+                value: this.disabled
```

Comment:
At this point the `this.disabled` is already updated?

Reply:
yes.